src/404.html

```diff
@@ -0,0 +1,30 @@
+---
+layout: "layout"
+class: 404
+banner: 404
+title: "404"
+description: "The page you were looking for doesn't exist"
+permalink: 404.html
+---
+
+<div class="grid">
+  <section class="grid-wide">
+    <h1>
+      Are you lost?
+    </h1>
+    <p>
+      Maybe we can help find what you're looking for.
+    </p>
+    <p>
+      You can get started using the Mythic editor by visting the <a href="https://app.mythic.nz/">dashboard.</a>
+    </p>
+    <p>
+      Need a hand? Take a look at our <a href="https://app.mythic.nz/help">help scrolls</a> for information on how to
```

Comment:
```suggestion
      Need a hand? Our <a href="https://mythic.nz/help">help scrolls</a> have information on how to
```

fyi help has moved to the marketing site